Src/VimCore/Interpreter_Interpreter.fs

```diff
@@ -644,6 +644,43 @@ type VimInterpreter
     member x.RunFunction (func: Function) = 
         _statusUtil.OnError Resources.Interpreter_FunctionNotSupported
 
+    /// Run the ':digraphs' command
+    member x.RunDigraphs digraphList =
+
+        // Get the two characters that make up the digraph.
+        let getChars (keyInputSet: KeyInputSet) =
+            keyInputSet.KeyInputs
+            |> Seq.map (fun keyInput -> string(keyInput.Char))
+            |> String.concat ""
+
+        // Get the display string for the replacement text.
+        let getDisplay (keyInputSet: KeyInputSet) =
+            getChars keyInputSet
+            |> StringUtil.GetDisplayString
+
+        // Get the unicode code point for the replacement text.
+        let getCode (keyInputSet: KeyInputSet) =
+            System.Char.ConvertToUtf32(getChars keyInputSet, 0)
+
+        if List.isEmpty digraphList then
+            let columns = 4
+            _keyMap.GetKeyMappingsForMode KeyRemapMode.Digraph
+            |> Seq.map (fun keyMapping ->
+                (
+                    getChars keyMapping.Left,
+                    getDisplay keyMapping.Right,
+                    getCode keyMapping.Right

```

Comment:
This feels like we are working against `KeyInputSet` here vs. it being of value. I think it would be a bit cleaner if we just held diagraphs in a separate map with a type that has the concrete behaviors we need: two characters and a code point.

Reply:
Yeah, kinda like vim storing a macro in a register.  :-)

When I was prototyping, it wasn't clear to me that digraphs *wouldn't* be used as real mappings in insert mode when `digraph` is set. And of course I was trying to leverage existing infrastructure to complete the job quickly. But I agree that a digraph-specific data structure that is a thin wrapper around a string -> string dictionary would be a better fit.